api/catalog/api/constants/categories.py

```diff
@@ -0,0 +1,35 @@
+class Categories(set):
+    def make_help_text(self) -> str:
```

Comment:
This is partially duplicated here:

https://github.com/WordPress/openverse-api/blob/f2ff9ae525ae060b65608b08f34d6fbc193d4346/api/catalog/api/serializers/media_serializers.py#L68

Reply:
Yours is a better implementation, might want to delete usage of this^ one.